django_select2/forms.py

```diff
@@ -228,6 +243,10 @@ def set_to_cache(self):
 
     def render_options(self, choices, selected_choices):
         """Render only selected options."""
+        if choices:
+            choices = [(k, self.label_from_instance(v)) for k, v in choices]
+        if self.choices:
```

Comment:
why is this needed? This will result in a massive query. The whole result set will be evaluated. No can do, sorry ;)